src/app/forms/controls/checkbox.html

```diff
@@ -0,0 +1,154 @@
+        <label class="clr-control-label">Basic</label>
+        <div class="clr-control-container">
+            <div class="clr-checkbox-wrapper">
+                <input type="checkbox" id="basic" class="clr-checkbox">
+                <label for="basic" class="clr-control-label">Label</label>
+            </div>
+            <clr-icon class="clr-validate-icon" shape="exclamation-circle"></clr-icon>
+            <span class="clr-subtext">Helper Text</span>
+        </div>
+    </div>
+
+    <div class="clr-form-control">
+        <label class="clr-control-label">Multi</label>
+        <div class="clr-control-container">
+            <div class="clr-checkbox-wrapper">
+                <input type="checkbox" id="multi-1" class="clr-checkbox">
+                <label for="multi-1" class="clr-control-label">Label</label>
+                <input type="checkbox" id="multi-2" class="clr-checkbox">
+                <label for="multi-2" class="clr-control-label">Label</label>
+                <input type="checkbox" id="multi-3" class="clr-checkbox">
+                <label for="multi-3" class="clr-control-label">Label</label>
+                <input type="checkbox" id="multi-4" class="clr-checkbox">
+                <label for="multi-4" class="clr-control-label">Label</label>
+                <input type="checkbox" id="multi-5" class="clr-checkbox">
+                <label for="multi-5" class="clr-control-label">Label</label>
+                <input type="checkbox" id="multi-6" class="clr-checkbox">
+                <label for="multi-6" class="clr-control-label">Label</label>
+            </div>
+            <clr-icon class="clr-validate-icon" shape="exclamation-circle"></clr-icon>
+            <span class="clr-subtext">Helper Text</span>
+        </div>
+    </div>
+
+    <div class="clr-form-control">
+        <label class="clr-control-label">Multi inline</label>
+        <div class="clr-control-container">
+            <div class="clr-checkbox-wrapper clr-checkbox-inline">
+                <input type="checkbox" id="multi-inline-1" class="clr-checkbox">
+                <label for="multi-inline-1" class="clr-control-label">Label</label>
+                <input type="checkbox" id="multi-inline-2" class="clr-checkbox">
+                <label for="multi-inline-2" class="clr-control-label">Label</label>
+            </div>
+            <clr-icon class="clr-validate-icon" shape="exclamation-circle"></clr-icon>
+            <span class="clr-subtext">Helper Text</span>
+        </div>
+    </div>
+
+    <div class="clr-form-control">
+        <label class="clr-control-label">Multi wrappers</label>
+        <div class="clr-control-container">
+            <div class="clr-checkbox-wrapper">
+                <input type="checkbox" id="multi-wrappers-1" class="clr-checkbox">
+                <label for="multi-wrappers-1" class="clr-control-label">Label</label>
+            </div>
+            <div class="clr-checkbox-wrapper clr-error">
```

Comment:
I see that you are using `.clr-error` here but the demo doesn't reflect that.
![image](https://user-images.githubusercontent.com/1426805/38572960-ac1d2fe6-3cc2-11e8-82d1-057c5a471ce8.png)

Reply:
You have to put `clr-error` on a `clr-control-container` for it to work, so this was a typo I fixed up. The example following is correct, since it shows the full container.